.github/workflows/build_windows.yaml

```diff
@@ -0,0 +1,56 @@
+name: Build standalone tool for Windows
+
+on: 
+  push:
+    branches:
+      - master
+
+
+jobs:
+
+  Compile-on-Windows:
+
+    runs-on: windows-latest
+    
+    if: github.event_name == 'push' && startsWith(github.ref, 'refs/tags')
```

Comment:
it would be useful to actually have the binaries ready also for intermediate commits, not just tags, so that people can test unreleased code as well